lisa/analysis/rta.py

```diff
@@ -229,11 +301,83 @@ def df_rtapp_stats(self, task=None):
         df = self._get_stats()
         return self._task_filtered(df, task)
 
+    @memoized
+    @df_rtapp_stats.used_events
+    def df_phases(self, task):
```

Comment:
Maybe a more descriptive name like df_task_phase_windows ?

Reply:
No, this returns a table:
```
|Time|duration|
```
which is the start and duration of each phase for the specified task.

Thus, `df_phases(task_name)` is exactly what it returns, windows need to be computed if required.